component/src/main/java/org/wso2/extension/siddhi/io/email/sink/EmailSink.java

```diff
@@ -52,97 +52,99 @@
-                + " (Refer link: https://javaee.github.io/javamail/SMTP-Transport to more information about"
-                + " smtp server parameters).",
+        description = "The email sink uses `smtp` server to publish events via emails. The format of the events can " +
+                "be `text`, `xml`, or `json`. You can configure parameters for an email sink in the stream definition" +
+                " or the `<SP_HOME>/conf/deplyment.yaml` file. The email source first checks the stream definition" +
+                " for parameters. If no parameters are configured there, the email source checks the " +
+                "`deployment.yaml` file. If the parameter configurations are not available in either place, the " +
+                "default values are considered (i.e., if they are available). If you need to configure server " +
+                "system parameters that are not provided as options in the stream definition, those should be " +
+                "defined in the `yaml`file under email sink properties. " +
+                "For more information about SMTP server parameters, see " +
+                "[SMTP Transport] (https://javaee.github.io/javamail/SMTP-Transport)",
+
         parameters = {
                 @Parameter(name = "username",
-                           description = "Username of the email account which is used to send emails"
-                                   + " (e.g: 'abc' is the username for abc@gmail.com).",
+                           description = "Username of the email account that is used to send emails. e.g., 'abc' is" +
+                                   " the username of `abc@gmail.com`).",
                            type = {DataType.STRING}),
                 @Parameter(name = "address",
-                           description = "Address of the email account which is used to send emails.",
+                           description = "The address of the email account that is used to send emails.",
                            type = {DataType.STRING}),
                 @Parameter(name = "password",
-                           description = "Password of the email account.",
+                           description = "The password of the email account that is used to send emails.",
                            type = {DataType.STRING}),
                 @Parameter(name = "host",
-                           description = "Host name of the smtp server "
-                                   + "(e.g. host name for a gmail account : 'smtp.gmail.com'). The default value"
-                                   + " 'smtp.gmail.com' is only valid if email account is a gmail account.",
+                           description = "The host name of the smtp server. The email address should always be " +
+                                   "`<USERNAME>@<HOST_NAME>.com'. (e.g., If `gmail` is the host name, the " +
```

Comment:
<USERNAME>@<HOST_NAME>.com is not correct.  As an example if host name for gmail account for smtp server is smtp.gmail.com